### DISCUSSION TITLE
Use `clap` for `helix-term`?

### DISCUSSION THREAD

**Discussion by TornaxO7** [helix-editor/helix#8830] [General] [answered]
Currently a lot of stuff is self written for the cli in `helix-term`. Would you be interested in a PR where [`clap`](https://crates.io/crates/clap) is used instead of the self implementation of [`helix-term/src/args.rs`](https://github.com/helix-editor/helix/blob/master/helix-term/src/args.rs)?

**Comment by the-mikedavis** [comment, marked as answer]
No, clap is a really heavy dependency for something we already solve minimally: `args.rs` is ~150 lines. We'd like to keep the number of CLI flags small since Helix is interactive so I don't think we need to invest into CLI flag parsing with any new crates.

**Reply by TornaxO7** [reply]
ok
